augurpy/read_load.py

```diff
@@ -30,26 +30,17 @@ def load(
         out = input
 
     elif isinstance(input, DataFrame):
-        try:
-            cell_type = input[cell_type_col]
-            label = input[label_col]
-        except KeyError:
-            print("[bold yellow]No column names matching cell_type_col and label_col. Looking in meta data.")
+        if meta is None:
             try:
-                cell_type = meta[cell_type_col]
-                label = meta[label_col]
-            except (KeyError, TypeError):
-                print("Missing label and / or cell type column.")
-                raise
+                cell_type = input[cell_type_col]
+                label = input[label_col]
+            except KeyError:
+                print("[bold red]No column names matching cell_type_col and label_col.")
```

Comment:
For the coloring to work we need

`from rich import print` at the top of the file :)